Gemfile

```diff
@@ -62,6 +62,10 @@ end
 
 group :test do
   # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
-  gem "capybara"
-  gem "selenium-webdriver"
+  gem 'capybara'
+  gem 'selenium-webdriver'
 end
+
+gem 'rack-cors'
+
+gem 'rubocop'
```

Comment:
**OPTIONAL** - You have done well so far, but there are some issues still on your project, according to the project requirements, you should set up your project with Webpack. Please add the `gem webpacker` to your Gemfile and run the following command `gem webpacker:install`. This will automatically generate some needed files for your project, in your case please overwrite all necessary files 👍 👍